common/test/acceptance/tests/lms/test_lms_dashboard.py

```diff
@@ -436,3 +450,44 @@ def test_dashboard_course_listings_a11y(self):
         course_listings = self.dashboard_page.get_courses()
         self.assertEqual(len(course_listings), 3)
         self.dashboard_page.a11y_audit.check_for_accessibility_errors()
+
+
+class TestMasqueradeAndSwitchCourse(BaseLmsDashboardTestMultiple):
+    """
+    Class to test lms dashboard accessibility of courses when masquerading as learner.
+    """
+
+    def test_masquerade_and_switch_course(self):
+        """
+        Scenario:
+            Masquerade as student in one course,
```

Comment:
We usually write scenarios as a series of steps starting with words like THEN, WHEN, AND, IF etc. See this file as an example https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/studio/test_studio_outline.py#L1018